### ISSUE TITLE
Remove option from the options array when added (tokenizer)

### ISSUE THREAD

**Issue by soen** [fmoo/react-typeahead#54]
Currently I'm able to search for an option already added, although I cannot add it twice. Is there a solution to hide an option from the options array, once it has been added?

**Comment by thehuey** [comment]
Not that I know of currently but sounds like a great feature to put in.  I can't think why it should not be the default behavior.

**Comment by fmoo** [comment]
Yeah, there's actually at least one class state quirk in the UI the current behavior causes.  The current behavior is kind of painful if you're trying to select all values that have a common prefix or something.

**Comment by growlsworth** [comment]
+1. I could see a use case for adding the same value multiple times, but I think the default use case should be to not return results that have already been selected.
